python/doc/theory/probabilistic_modeling/conditional_distributions.rst

```diff
@@ -0,0 +1,457 @@
+.. _conditional_distributions:
+
+Conditional distributions
+-------------------------
+
+The library offers some modelisation capacities on conditional distributions:
+
+- Case 1: Create a joint distribution from its conditioned margins,
+- Case 2: Create a distribution which parameters are aleatory,
+- Case 3: Condition a joint distribution by some values of its margins,
```

Comment:
Maybe we could swap cases 1 & 3 ? Feels current case 3 would be more frequently needed dont you think ?

Reply:
yes. You are right.